odxtools/positioneddataobjectproperty.py

```diff
@@ -0,0 +1,74 @@
+# SPDX-License-Identifier: MIT
```

Comment:
as said above, I'd prefer to cling to the XSD and duplicate this class into `SwitchKey` and `DetermineNumberOfItems`